src/transaction/log_impl.h

```diff
@@ -759,6 +760,85 @@ typedef struct log_logging_stat
   unsigned long async_commit_request_count;
 } LOG_LOGGING_STAT;
 
+/*for CDC interface */
```

Comment:
Could you please separate the structures, defines, etc related to the CDC operation?

Reply:
Additionally, I think you should use/make the prefix for CDC. It makes it easy for other developers to understand your code and to sustain your code easily.

Reply:
OK, I separated them, and modified names of data structures and  functions for more readability.